isb_web/templates/thing.html

```diff
@@ -23,6 +23,61 @@
   </link>
   <script type="module"
     src="https://cdn.jsdelivr.net/gh/isamplesorg/isamples_webui@develop/record-page/src/js/index.js"></script>
+  <script src="https://hypothes.is/embed.js" async></script>
+  <script>
+async function fetchGrantToken(authority, isamples_jwt_url, hypothesis_api_url, login_url, logout_url) {
+  var grantToken = "";
+  fetch(isamples_jwt_url, {
+    'method': 'POST',
+    credentials: "include",
```

Comment:
This is the magic sauce to include get the session cookie included in the JS request.  This, combined with the CORS changes, allowed us to properly access the logged-in session.